R/tags.R

```diff
@@ -268,7 +268,7 @@ normalizeText <- function(text) {
 #' @examples
 #' tagList(
 #'   h1("Title"),
-#'   h2("Header text"),
+#'   h2("Header text", style = "text-align: center;"),
```

Comment:
```suggestion
#'   h2("Header text", style = "color: red;"),
```

Small nit that setting text color is a more likely usage for inline styles than center-alining an `<h2>`.

Reply:
Good idea!